docs/getting-started/supported-languages-and-tools.md

```diff
@@ -265,13 +287,21 @@ The table below lists all languages that Codacy supports and the corresponding t
       <td><a href="https://github.com/detekt/detekt">detekt</a>,
           <a href="https://semgrep.dev/">Semgrep</a> <a href="#semgrep"><sup>1</sup></a></td>
       <td>-</td>
-      <td>-</td>
+      <td><a href="https://semgrep.dev/">Semgrep</a></td>
       <td>-</td>
       <td><a href="https://github.com/kucherenko/jscpd">jscpd</a></td>
       <td><a href="https://github.com/detekt/detekt">detekt</a></td>
     </tr>
     <tr>
       <td>Kubernetes</td>
+      <!--
```

Comment:
to uncomment this we need to first add Language.YAML as a supported language to Semgrep in codacy/codacy-tools